web/src/theme/prism-include-languages.js

```diff
@@ -0,0 +1,22 @@
+import siteConfig from "@generated/docusaurus.config";
```

Comment:
This file gets autogenerated when you "eject" to add custom languages to Docosaurus

Reply:
Probably best to add this comment directly to this file!

Reply:
I'll move it there 👍